index.md

```diff
@@ -30,6 +30,8 @@ To find out more about a specific system and a download link, follow one of the
 * [clingcon](/clingcon/) extends clingo with constraint solving capabilities.
 * [aspcud](/aspcud/) is a solver for package dependencies.
 * [asprin](/asprin/) is a general framework for qualitative and quantitative optimization in ASP.
+* [clingraph](/clingraph/) is a tool for visualizing ASP encodings using ASP.
```

Comment:
Maybe write 
```
is a visualization tool for ASP
```
to avoid the double ASP

Reply:
Could be "is a tool to define graphical visualization in ASP" because the magic is that the visualization is in ASP :)

Reply:
It still sounds clumsy. :smile: The graphical sounds redundant because I would think that visualization is always graphical.

Reply:
I don't think we have to be to precise here. 

What about just telling what it can do
```
is a tool to visualize answer sets
```
The reader can figure out the how by following the link.